### PR TITLE
Allow display of zero values in DetailView

### PR DESCRIPTION
**Pull request by beastbytes**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #90

### CODE REVIEW

File: src/DataAttribute.php [hunk 1]
```diff
@@ -139,7 +139,7 @@ public function getValue($model): string
             $value = ArrayHelper::getValueByPath($model, $this->name);
         }
 
-        if (empty($value)) {
+        if (empty($value) && (int)$value !== 0) {
```
Comment (vjik) [inline src/DataAttribute.php:142]:
```suggestion
        if ($value === null || $value === '') {
```

Seems, it's more clearly.

Comment (alamagus) [reply src/DataAttribute.php:142]: `if (empty($value) && (int)$value !== 0)` === `if (false)`
If you're gonna rewrite as @vjik suggested, don't forget about other 'empty' values like `[]`, mb `false`

Comment (vjik) [reply src/DataAttribute.php:142]: Agree with empty array. 

For boolean values maybe print words "true" and "false" ?

Comment (terabytesoftw) [reply src/DataAttribute.php:142]: I could finish adopting the suggestions to merge it, thanks.

**Pull request closed without merging**